lib/rialto/etl/configs/stanford_organizations_to_sparql_statements.rb

```diff
@@ -49,10 +49,14 @@ def contextualized_organization_name(organization)
 }
 
 # Org label
-to_field '!label', literal(true)
-to_field '@label', lambda { |json, accum|
-  accum << contextualized_organization_name(json)
-}, single: true
+to_field '!' + SKOS['prefLabel'].to_s, literal(true)
```

Comment:
Since we are using string interpolation (vs. concatenation) elsewhere in this branch, how would you feel about changing that in this file? (If you feel it should be follow-up work, that's cool too.)